CONTRIBUTING.md

```diff
@@ -108,6 +108,26 @@ to make that happen.
 
 Happy developing!
 
+### Using DummyAuthenticator & SimpleSpawner
+
+The default JupyterHub authenticator & spawner require your
+system to have user accounts for each user you want to log in to
+JupyterHub as. This requires you have root / `sudo` access to your
```

Comment:
Perhaps add a link to the default authenticators / spawners as this text may raise the question about what those are called. I figure a link would be less intrusive to the narration than dropping names of stuff.

> The default JupyterHub [authenticator](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html#the-default-pam-authenticator) & [spawner](https://jupyterhub.readthedocs.io/en/stable/api/spawner.html#localprocessspawner) require your
system to have user accounts for each user you want to log in to
JupyterHub as.

---

I'm not sure, but one could consider adding a one to three sentences before this clarifying the point of an authenticator and spawner. Perhaps the links are enough though, yeah, probably, yepp, lets not repeat ourselves too much.

Reply:
I agree! Will do when I'm at a computer :) feel free to push a change too if you wanna